documentation/book/con-reassignment-throttles.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// assembly-scaling-clusters.adoc
+
+[id='con-reassignment-throttles-{context}']
+
+= Reassignment throttles
+
+Reassigning partitions can be a slow process because it can require moving lots of data between brokers.
+To avoid this having a detrimental impact on clients it is possible to _throttle_ the reassignment.
```

Comment:
To avoid a detrimental impact on clients, you can throttle the reassignment process. This might cause the reassignment to take longer to complete.